components/network/asterisk/files/asterisk.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" ?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<service_bundle type="manifest" name="asterisk">
+    <service name="network/asterisk" type="service" version="0">
+	<create_default_instance enabled="false"/>
+
+        <dependency name="network" grouping="require_all" restart_on="error" type="service">
+            <service_fmri value="svc:/milestone/network:default"/>
+        </dependency>
+
+        <dependency name="filesystem" grouping="require_all" restart_on="error" type="service">
+            <service_fmri value="svc:/system/filesystem/local"/>
+        </dependency>
+
+        <method_context>
+            <method_environment>
+                <envvar name="PATH" value="/usr/sbin"/>
+            </method_environment>
+        </method_context>
+
+        <exec_method type="method" name="start" exec="/usr/sbin/asterisk -U asterisk -G asterisk" timeout_seconds="60"/>
```

Comment:
Do we want to use -U -G options or <method_credential user='asterisk' group='asterisk'/>